### DISCUSSION TITLE
ARCore 이미지 인식 질문

### DISCUSSION THREAD

**Discussion by hemil0102** [hemil0102/GumamdongAR#6] [Q&A]
ARCore를 활용한 이미지 인식을 구현하려고 했을 때, 
구글 문서를 살펴보니 iOS 환경에서 ARCore 이미지 인식 API가 보이지 않습니다. 
예제들을 살펴봐도 없어보이는데, 

ARCore는 iOS에 Geospatial, Cloud Anchor, Augmented Face 관련 기능을 제공하고, 
ARKit에서 가능한 부분은 구현 안해놓은 것 같은데요. 

따라서 이미지 인식 자체는 ARKit을 통해서 ImageAnchor를 통해 구현해야 할 것 같은데, 
자료를 찾아보고 있는데 제가 이해하고 있는게 맞을까요?

**Comment by jongfeel** [comment]
Image anchor는 arkit으로 해도 될 것 같습니다.
저도 사실 iOS에서 arcore를 써서 image anchor를 써보진 않았는데
API를 봤을 때는 지원 안하는 것 같네요

https://developers.google.com/ar/reference/ios
